randovania/games/planets_zebeth/logic_database/Brinstar.json

```diff
@@ -1806,12 +1808,29 @@
                     "override_default_lock_requirement": null,
                     "connections": {
                         "Door to Corridor 7": {
-                            "type": "resource",
+                            "type": "and",
                             "data": {
-                                "type": "items",
-                                "name": "Ice Beam",
-                                "amount": 1,
-                                "negate": false
+                                "comment": null,
+                                "items": [
+                                    {
+                                        "type": "resource",
+                                        "data": {
+                                            "type": "items",
+                                            "name": "Ice Beam",
+                                            "amount": 1,
+                                            "negate": false
+                                        }
+                                    },
+                                    {
+                                        "type": "resource",
+                                        "data": {
+                                            "type": "tricks",
+                                            "name": "IceAbuse",
+                                            "amount": 1,
+                                            "negate": false
+                                        }
+                                    }
+                                ]
```

Comment:
I would put it at least Intermediate because of bad RNG. It can take up to 30 mins depending on the RNG for now.

Reply:
30 min sounds way worse than just intermediate lol